PurchasesTests/Identity/IdentityManagerTests.swift

```diff
@@ -66,6 +66,18 @@ class IdentityManagerTests: XCTestCase {
         assertCorrectlyIdentified(newManager, expectedAppUserID: newAppUserID)
     }
 
+    func testNilAppUserIDBecomesAnonimous() {
+        assertCorrectlyIdentifiedWithAnonymous(create(appUserID: nil))
+    }
+
+    func testEmptyAppUserIDBecomesAnonymous() {
+        assertCorrectlyIdentifiedWithAnonymous(create(appUserID: ""))
+    }
+
+    func testEmptyAppUserWithSpacesIDBecomesAnonymous() {
+        assertCorrectlyIdentifiedWithAnonymous(create(appUserID: "  "))
+    }
+
```

Comment:
Perhaps we should capture the new behavior with a test that checks `" nacho with spaces "` works and doesn't trim leading and trailing spaces

Reply:
👍  done
Not that this isn't a "new" behavior though, just pointing out that I didn't change it :)